<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>u=</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="uLine">
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>x0 = </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="x0Line">
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Посчитать</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>78</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
   </layout>
   <zorder>x0Line</zorder>
   <zorder>uLine</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>gLine</zorder>
   <zorder>label_3</zorder>
   <zorder>uLine</zorder>
   <zorder>x0Line</zorder>
   <zorder>uLine</zorder>
   <zorder>pushButton</zorder>
   <zorder>textBrowser</zorder>
   <zorder>horizontalSpacer</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
